tutor/specs/components/new-course/build-course.spec.cjsx

```diff
@@ -12,13 +13,13 @@ describe 'CreateCourse: saving new course', ->
     NewCourseActions.save.restore()
 
   it 'it transitions to ready after save', ->
-    wrapper = shallow(<BuildCourse />)
+    wrapper = mount(<BuildCourse />)
     expect(NewCourseActions.save.calledOnce).to.be.true
     expect(wrapper.text()).to.include('building')
-    expect(wrapper.find('Icon[type="refresh"]')).to.have.length(1)
-    NewCourseActions.created({id: 1})
+    expect(wrapper.find('.ox-loader')).to.have.length(1)
+    NewCourseActions.created({id: '1'})
     wrapper.setState({})
-    expect(wrapper.find('Icon[type="refresh"]')).to.have.length(0)
+    expect(wrapper.find('.ox-loader')).to.have.length(0)
     expect(wrapper.text()).to.include('course is ready')
```

Comment:
FYI, it's much faster to use `shallow` vs `mount`, the only problem with it is that you can't actually test the dom for nested components since they don't render.  You can test the component itself using enzyme's finders though.  You could have tested something like:

`expect(wrapper.find('OXFancyLoader[isLoading=true]')).to.have.length(1)`